src/lib/markup/photodiode.js

```diff
@@ -4,29 +4,42 @@ import { eventCodes } from "../../config/trigger";
 import { div, span } from "./tags";
 
 /**
- * Displays a box in the bottom right corner of the screen with the id "photodiode-spot"
+ * Markup for a box in the bottom right corner of the screen and a photodiode spot inside the ghost box
+ * Note the box will only be visible if USE_PHOTODIODE is true
  */
-function photodiodeGhostBox() {
-  const spot = span("", { id: "photodiode-spot", class: "photodiode-spot" });
-  return div(spot, {
-    id: "photodiode-box",
-    // Photodiode is only visible if config.USE_PHOTODIODE is true
-    class: config.USE_PHOTODIODE ? "photodiode-box visible" : "photodiode-box invisible",
-  });
-}
+const photodiodeGhostBox = div(span("", { id: "photodiode-spot", class: "photodiode-spot" }), {
+  id: "photodiode-box",
+  class: config.USE_PHOTODIODE ? "photodiode-box visible" : "photodiode-box invisible",
```

Comment:
Is `config` guaranteed to be defined when this is run? It always makes me a little nervous to put conditional stuff at the top level.

Reply:
It is, but I agree I'd like to change it at some point.